### PR TITLE
Calculate Packet Loss Statistics

### PR DESCRIPTION
**Pull request by samgomena**
Per discussion on #58 this PR adds:
* A method on the `ResponseList` object that returns the percent of packets lost where 0% packets lost = a return value of `0.0` and 100% packet loss = a return value of `1.0`. The number is calculated as a running average so no additional computation is done when the method is called.

I didn't add this stat to the `__repr__` method, but would be happy to if you think it would be useful!

Thanks for reviewing in advance!

### CODE REVIEW

**Comment by alessandromaggio** [comment]
Just had a quick review and the code looks good, but it lacks testing. @samgomena can you please add unit testings for this feature?
